packages/react-vapor/src/utils/FalsyValuesUtils.spec.tsx

```diff
@@ -39,7 +39,7 @@ describe('callIfDefined', () => {
     });
 
     it('should not throw errors when calling with undefined values', () => {
-        const someDeclaredButNotAssignedCallback: () => any = undefined;
+        const someDeclaredButNotAssignedCallback: any = undefined;

```

Comment:
I think `const someDeclaredButNotAssignedCallback: () => any;` would be better to reflect what we actually want to validate

Reply:
If you activate `strictNullCheck` in this repo, this line breaks a compilation error when you keep the type  :P

I mean... This line gets compiled to `var someDeclaredButNotAssignedCallback = undefined` either way, so I'm not sure what it is actually testing

Reply:
You're right, it's only testing that typescript work as expected 😛